iDroidLayout/Widgets/Categories/UIScrollView+IDL_ViewGroup.m

```diff
@@ -231,7 +231,7 @@ - (void)onLayoutWithFrame:(CGRect)frame didFrameChange:(BOOL)changed {
             maxY = MAX(maxY, childTop + height);
         }
     }
-    self.contentSize = CGSizeMake(maxX + padding.right, maxY + padding.bottom);
+    self.contentSize = CGSizeMake([self.subviews count] * (maxX + padding.right), maxY + padding.bottom);
```

Comment:
Okay, I would suggest making the change specifically to NSInteger count = self.subviews.count instead and eliminating the change you made. 

Your change assumes the width is the same for every child and it incorrectly multiplies the parent's right padding for every child.